src/patternfly/components/DataList/data-list.scss

```diff
@@ -10,9 +10,9 @@ $pf-data-list--breakpoint--mobile: 380px;
   --pf-c-data-list__item--mobile--PaddingRight: var(--pf-global--spacer--lg); // custom name for one-off breakpoint
   --pf-c-data-list__item--PaddingLeft: var(--pf-global--spacer--xl);
   --pf-c-data-list__item--mobile--PaddingLeft: var(--pf-global--spacer--lg);
```

Comment:
Pretty sure we used responsive instead of mobile in other places

Reply:
Ah, I see this in the table component. Though it's odd, the breakpoint in that component is called `$pf-c-table-xs-breakpoint`, but the `--pf-c-table-td--responsive*` classes don't apply to that breakpoint. I'm fine with `--mobile` here, or maybe we call it `$pf-data-list--breakpoint--data-table-xs` with vars like `--pf-c-data-list__item--data-table-xs--PaddingRight`. I dunno, what do you think?

Reply:
Hm I don't mind, we can keep mobile here!